Lib/fontTools/feaLib/lexer.py

```diff
@@ -45,13 +47,13 @@ def __init__(self, text, filename):
     def __iter__(self):
         return self
 
-    def next(self):  # Python 2
-        return self.__next__()
+    def next(self, comments = False):  # Python 2
+        return self.__next__(comments)
 
-    def __next__(self):  # Python 3
+    def __next__(self, comments = False):  # Python 3
         while True:
             token_type, token, location = self.next_()
-            if token_type not in {Lexer.COMMENT, Lexer.NEWLINE}:
+            if token_type not in (Lexer._OMITS if comments else Lexer._OMITC):
```

Comment:
This might be a bit easier to read? Then you don’t need such cryptic constants.

```python
if token_type == COMMENT and not comments:
    continue
if token_type == Lexer.NEWLINE:
    continue
```

Or perhaps:

```python
ignore = ((token_type == Lexer.NEWLINE) or
          (token_type == Lexer.COMMENT and not comments))
if not ignore:
    return (token_type, token, location)
```